docs/submit-opentargets-batch.md

```diff
@@ -242,8 +265,8 @@ sources (EVA, Open Targets, GWAS, Uniprot). In order to do so, execute the follo
 
 ```bash
 python bin/clinvar_jsons/traits_to_zooma_format.py \
```

Comment:
you added the `bsub` part to other commands. why not to this one? is it just a lightweight process?

Reply:
No, I just forgot about this one. Fixed, thanks!